docs/kubernetes/user-guides/pre-provisioned.md

```diff
@@ -0,0 +1,63 @@
+# Kubernetes Pre-Provisioned Disks User Guide
```

Comment:
Do we have some Table of Contents page that can link to all these user-guides?

Reply:
There's a link to the "user guides" folder from the main README